style.css

```diff
@@ -291,18 +256,6 @@ ul {
   list-style: none;
```

Comment:
- [ ] Great job. everything works fine. please kindly add little margin left and right to your contact text section
- [ ] please use thesame figma color for your social icons so it follows the figma design

yours
![Screenshot (476)_LI](https://user-images.githubusercontent.com/65298048/192746369-c11de177-c063-40da-a4f6-77d4309f9860.jpg)
![Screenshot (477)_LI](https://user-images.githubusercontent.com/65298048/192746401-752be116-2f5a-4eea-9b2c-06f1df5e8b69.jpg)

figma

![Screenshot (478)](https://user-images.githubusercontent.com/65298048/192746452-036f8098-03a4-4f54-b7f8-39a391a8cf4d.png)